GenericTypeType.Tests/WorkerManagerGetTypePatchTests.cs

```diff
@@ -20,7 +20,9 @@ public HarmonyRewriteSetup()
   }
 }
 
-public class WorkerManagerGetTypePatchTests : IClassFixture<HarmonyRewriteSetup>
+#pragma warning disable xUnit1033 // Test classes decorated with 'Xunit.IClassFixture<TFixture>' or 'Xunit.ICollectionFixture<TFixture>' should add a constructor argument of type TFixture
+public class WorkerManagerGetTypePatchTests : IClassFixture<HarmonyRewriteSetup>

```

Comment:
Does this test still work?

Reply:
Test should still work. I ran the unit tests and made sure they succeeded before making the PR.